src/main/java/frc/robot/subsystems/endEffector/EndEffectorIO.java

```diff
@@ -1,12 +1,12 @@
 package frc.robot.subsystems.endEffector;
 
-interface EndEffectorIO {
+public interface EndEffectorIO {
```

Comment:
Same thing with what I was talking about earlier. If the way I suggested works, then we don't need it to be public.

It was meant to be a temp workaround to begin with